Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3247,7 +3247,7 @@ class BlurWrapper: UIView {
     }
     
 
-    private var effectView: UIVisualEffectView
```

Comment:
Oh, no — I changed that for a previous approach that didn't work so well, and must have forgotten it wasn't used any more!